services-directions-models/src/main/java/com/mapbox/api/directions/v5/DirectionsCriteria.java

```diff
@@ -416,4 +416,47 @@ private DirectionsCriteria() {
   })
   public @interface ApproachesCriteria {
   }
+
+  // EXPERIMENTAL
```

Comment:
Have we always highlighted experimentation API by a comment? Maybe it's better to add an 'Unstable' or 'Experimental' annotation like [this](https://stackoverflow.com/a/32616239/3599375) ?

Reply:
This particular comment is just for us to organize the code, unless you mean all the other actual javadoc comments?

There's no precedent for experimental mapbox-java APIs and I quickly tested out adding a custom annotation that uses `@Documented` but it didn't do the trick, javadoc still requires me to write documentation for the annotation method.

Reply:
Or are you thinking about a supplement in form of annotation, next to javadoc comments saying that the method is experimental?

Reply:
I was thinking about an `Experimental` annotation which will be visible in java-docs just as annotation. It could be a standard way to mark an API as not stable instead of manually writing messages about stability